be/src/Infrastructure/ApiExtensions/Foundation.ApiExtensions/Definitions/ApiConstants.cs

```diff
@@ -13,6 +13,11 @@ public struct Catalog
             public static readonly string ProductSuggestionsApiEndpoint = "/api/v1/productsuggestions";
         }
 
+        public struct Groups
+        {
+            public static readonly string GroupsApiEndpoint = "/api/v1/groups";
```

Comment:
**suggestions:** Create a 'Client' struct and put this and other client-related endpoints there